sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -986,6 +993,10 @@ def on_port_update_event(self, port_change_event):
                 self.port_dict[lport]['speed'] = port_change_event.port_dict['speed']
             if port_change_event.port_dict is not None and 'lanes' in port_change_event.port_dict:
                 self.port_dict[lport]['lanes'] = port_change_event.port_dict['lanes']
+            if port_change_event.port_dict is not None and 'host_tx_ready' in port_change_event.port_dict:
+                self.port_dict[lport]['host_tx_ready'] = port_change_event.port_dict['host_tx_ready']
+            if port_change_event.port_dict is not None and 'admin_status' in port_change_event.port_dict:
+                self.port_dict[lport]['admin_status'] = port_change_event.port_dict['admin_status']
             self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_INSERTED
```

Comment:
Change the CMIS state to inserted and invoke reset_cmis() only when there is a change in value for the following fields speed / lanes / host_tx_ready or admin_status.  
With current change, the cmis state is set to INSERTED even for oper_state state change in value in the DB for a Port.

Reply:
the port change event doesn't have a mechanism to filter out the event based upon the corresponding PORT_TABLE. So, until we have this mechanism to filter out, we need this code here otherwise Xcvrd will miss optics jack out/in